code/modules/reagents/chems/chems_drinks.dm

```diff
@@ -178,15 +178,16 @@
 	allergen_flags = ALLERGEN_VEGETABLE
 
 /decl/material/liquid/drink/juice/garlic
-	name = "garlic juice"
-	lore_text = "Who would even drink this?"
+	name = "garlic oil"
```

Comment:
clearly we need `lipophilicity` as a material var to generalize oils in the code (joke)